snapcraft/main.py

```diff
@@ -308,7 +308,13 @@ def run(args, project_options):  # noqa
         parts.define(args['<part-name>'])
     elif args['search']:
         parts.search(' '.join(args['<query>']))
+    elif os.environ.get('SNAPCRAFT_CONTAINER_BUILDS'):
```

Comment:
Just rediscovered `os.getenv` too, not sure which one is preferred. But I write here due to another reason, is the next PR going to consume this env globally and make all the commands pass into the ProjectContainer class so that I can do

`SNAPCRAFT_CONTAINER_BUILDS=1 snapcraft pull`
`SNAPCRAFT_CONTAINER_BUILDS=1 snapcraft build part-n`
`SNAPCRAFT_CONTAINER_BUILDS=1 snapcraft clean`

We probably need to guard (or ignore the variable) when using `cleanbuild`